component/avatar/avatar.css

```diff
@@ -0,0 +1,58 @@
+.avatar-container {
+  display: flex;
+  flex-direction: column;
```

Comment:
declare these properties also , and use it . same as flex-row.

Reply:
done